src/test/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSourceTest.java

```diff
@@ -0,0 +1,70 @@
+package io.jenkins.plugins.gitlabbranchsource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+
+import hudson.model.TaskListener;
+import hudson.security.AccessControlled;
+import hudson.util.StreamTaskListener;
+import io.jenkins.plugins.gitlabbranchsource.helpers.GitLabHelper;
+import io.jenkins.plugins.gitlabserverconfig.servers.GitLabServer;
+import io.jenkins.plugins.gitlabserverconfig.servers.GitLabServers;
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Set;
+import jenkins.branch.BranchSource;
+import jenkins.scm.api.SCMHead;
+import org.gitlab4j.api.GitLabApi;
+import org.gitlab4j.api.MergeRequestApi;
+import org.gitlab4j.api.ProjectApi;
+import org.gitlab4j.api.RepositoryApi;
+import org.gitlab4j.api.models.Project;
+import org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+public class GitLabSCMSourceTest {
+
+    private static final String SERVER = "server";
+    private static final String PROJECT_NAME = "project";
+    private static final String SOURCE_ID = "id";
+
+    @Rule
+    public final RestartableJenkinsRule plan = new RestartableJenkinsRule();
```

Comment:
Any particular reason to use `RestartableJenkinsRule` here over plain `JenkinsRule`?

Reply:
No particular reason. I changed the test to use JenkinsRule.